sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -85,15 +84,6 @@ public FormTrainingAsyncClient getFormTrainingAsyncClient() {
         return new FormTrainingAsyncClient(this.service, this.serviceVersion);
     }
 
-    /**
```

Comment:
Why is service version getting removed in this PR

Reply:
It happens to TA as well. "Unless there is a use case, for now lets leave these getServiceVersion calls out."
https://apiview.dev/Assemblies/Review/05d0102010704cfa977f9386ce622294/067d8625611f4b329856ba69b7774d8d